apps/fluent-tester/src/FluentTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Avatar, Size } from '@fluentui-react-native/experimental-avatar';
+import { Avatar, Size } from '@fluentui-react-native/experimental-avatar/src/Avatar.macos';
```

Comment:
nit: You can probably fork AvatarTest per platform too, and just have them test different cases